src/python/run_matching.py

```diff
@@ -33,10 +33,10 @@ def parseParams():
     return parser.parse_args()
 
 
-def setDictParam(args):
+def setDictParam(args, query_features_dir, reference_features_dir):
     params = dict()
-    params["path2qu"] = args.query_features.as_posix()
-    params["path2ref"] = args.reference_features.as_posix()
+    params["path2qu"] = query_features_dir.as_posix()
```

Comment:
Same as above about the `as_posix` part.

Reply:
Thanks, removed posix.